subscriptions/management/commands/fix_subscription_lifecycle.py

```diff
@@ -8,9 +8,11 @@
 
 class Command(BaseCommand):
     help = ("Fast forward subscriptions by one message if it is behind "
-            "schedule. This is used when the messages sending failed to get "
-            "the subscription up to date. Leave end_date blank for current "
-            "date. Include `--fix True` to update and send messages")
+            "schedule or fast forward to end date. This is used when the "
+            "messages sending failed to get the subscription up to date. Leave"
+            " end_date blank for current date. Include `--action fast_forward`"
+            " to fast forward them to the end date or `--action send` to send "
+            "messages")
```

Comment:
I think we might want to reword this so that it's clear that `fast_forward` is the one that fast forwards to `end_date`, and `send` is the one that sends one message if the subscription is behind.